### PR TITLE
manage link if url available in card content

### PR DESCRIPTION
**Pull request by Xb0X**

### CODE REVIEW

File: src/components/Card/ContentCard.js [hunk 1]
```diff
@@ -163,7 +164,7 @@ const ContentCard = ({ card, type, ...props }) => {
 ContentCard.propTypes = {
```
Comment (ldabiralai) [inline src/components/Card/ContentCard.js]: How can we make it clear that `linkComponent` is a prop here (and what that prop can/should be)

Comment (bdo) [reply src/components/Card/ContentCard.js]: Good point, this need to be documented in the README.md

Comment (bdo) [reply src/components/Card/ContentCard.js]: To clarify the purpose, it allows you to override the link behaviour in your app when you need to use a router link.

For instance if you wanted to use React Router for internal links, you could use something like think:
```js
import React from 'react';
import { Link as EurosportWebToolkitLink } from '@eurosport/web-toolkit';
import { Link as ReactRouterLink } from 'react-router-dom';

const isExternalUrl = url => url && url.match(/^(https?|mailto|tel):/);

EurosportWebToolkitLink.defaultProps.linkComponent = ({ href, children, ...props }) => {
  if (isExternalUrl(href)) {
    return (
      <a {...props} href={href}>
        {children}
      </a>
    );
  }
  return (
    <ReactRouterLink {...props} to={href}>
      {children}
    </ReactRouterLink>
  );
};

```

Comment (ldabiralai) [reply src/components/Card/ContentCard.js]: @bdo, I understand the purpose, but it's very hidden away. My concern is that it's not discoverable, how can we make it obvious?

Is the prop listed in the storybook documentation?

**Pull request merged**